app/src/androidTest/java/com/nervousfish/nervousfish/test/ContactActivitySteps.java

```diff
@@ -69,13 +92,69 @@ public void iPressDeleteButton() {
         onView(withId(R.id.deleteButton)).perform(click());
     }
 
+    @When("^I press that I am sure$")
+    public void iAmSureToDelete() {
+        onView(withId(R.id.confirm_button)).perform(click());
+    }
+
+    @When("^I press on the OK button$")
+    public void iPressOK() {
+        onView(withId(R.id.confirm_button)).perform(click());
+    }
+
     @Then("^I should go to the previous activity I visited$")
     public void iShouldGoToPreviousActivity() {
         assertEquals(LoginActivity.class, getCurrentActivity().getClass());
     }
 
-    @Then("^I should get a popup asking if I am sure to delete the contact$")
-    public void iShouldGetPopup() {
+    @Then("^the current contact should be deleted$")
```

Comment:
I don't think this the norm, "Then the current contact should be deleted" is a normal sentence and is wrong when The would start with an uppercase letter